tox.ini

```
@@ -48,6 +48,7 @@ commands =
 
 [testenv:integration]
 deps =
+    juju == 3.0.0
```

you can do this to pin to our libjuju

```suggestion
    git+https://github.com/charmed-kubernetes/python-libjuju.git@2.9.11+ck1 
```